Dockerfile

```diff
@@ -9,7 +9,11 @@ ENV DEBIAN_FRONTEND=noninteractive
 ENV LIBGL_ALWAYS_INDIRECT=1
 ENV PATH="${PATH}:/root/.gem/ruby/2.7.0/bin:/root/.local/share/gem/ruby/2.7.0/bin"
 
-RUN apt-get update && apt-get install -y build-essential python3-pip python3-dev libldap2-dev libsasl2-dev python3-pyqt5 ruby && \
+# This is needed to install qt6-base-dev on bullseye
+RUN echo "deb http://deb.debian.org/debian bullseye-backports main contrib non-free" >> /etc/apt/sources.list && \
+    echo "deb-src http://deb.debian.org/debian bullseye-backports main contrib non-free" >> /etc/apt/sources.list
+
+RUN apt-get update && apt-get install -y build-essential python3-pip python3-dev libxcb-cursor0 qt6-base-dev ruby && \
```

Comment:
I don't see a reason, why we would need `libldap2-dev libsasl2-dev`, also `python3-pyqt5` is actually Qt binding for PyQt5 (which is alternative Python binding to PySide2), we most likely were just fine installing `qtbase5-dev`, which before was installed as dependency of `python3-pyqt5`